db/migrate/20170410135556_add_csv_to_imports.rb

```diff
@@ -0,0 +1,6 @@
+class AddCsvToImports < ActiveRecord::Migration[5.0]
+  def change
+    add_column :imports, :csv, :string
+    remove_column :imports, :uri
```

Comment:
I wonder if renaming would be easier - technically, this migration is not reversible because the `remove_column` doesn't include the type.

One other thought - calling this column `csv` does lock us into a particular format. What was nice about naming the column `uri` is that it didn't imply any particular format. It could just as easily been JSON or XML or whatever.